refml/topic_modeling/refml_sql.py

```diff
@@ -1,12 +1,61 @@
 from refml.topic_modeling.preprocess import stem_and_tokenize
 from refml.topic_modeling.model import refml_nmf_tsne
 import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.types import Integer, String, Float
+import cx_Oracle
 
-def runrefml(input, n_topics, n_top_words, n_dimensions, perplexity):
-    df['Text_Processed'] = input.apply(stem_and_tokenize)
-    return refml_nmf_tsne(df.Text_Processed, n_topics, n_top_words, n_dimensions, perplexity)
+def hero_to_df(Userid, Password, Host, Port, SID, query):
+    dsn_tns = cx_Oracle.makedsn(host=Host, port=Port, sid=SID)
+    conn = cx_Oracle.connect(user=Userid, password=Password, dsn=dsn_tns)
 
-def refml_for_sql(sql_in, input_column, n_topics, n_top_words, n_dimensions, perplexity):
-    input = pd.from_sql(sql_in)
-    output = runrefml(input.input_column, n_topics, n_top_words, n_dimensions, perplexity)
-    return pd.to_sql(output)
\ No newline at end of file
+    return pd.read_sql_query(query, conn)
+
+def refml_to_sql(df, input_column, n_topics, n_top_words, n_dimensions, perplexity):
```

Comment:
let's refactor and call this `refml_nfm_tsne_to_db` refactor. could this method perhaps take the output of `refml_nmf_tsne` and a `conn` or an `engine` as as an input?.